doc/setup/install.rst

```diff
@@ -106,7 +106,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 11.0 'deb' package currently supports `Debian Stretch`_, `Ubuntu Xenial`_,
+Since Odoo 12.0 'deb' package supports `Debian Stretch`_, `Ubuntu Xenial`_,
```

Comment:
Don't use "since" here as it is likely to change in upper versions.
@d-fence xenial is the 16.04, shouldn't it be 18.04 (LTS as well) now? Is there all the min requirements in the 16.04?

Reply:
Sure, it will fail with Xenial packages. You need 18.04